server/app/services/applicant/ApplicantService.java

```diff
@@ -587,24 +594,33 @@ CompletionStage<ApplicationModel> submitApplication(
                       .orElse(CompletableFuture.completedFuture(null));
 
               ApplicantPersonalInfo applicantPersonalInfo = applicantLabelFuture.join();
-              Optional<String> applicantEmail =
+              Optional<ImmutableSet<String>> applicantEmails =
                   applicantPersonalInfo.getType() == ApplicantType.LOGGED_IN
                       ? applicantPersonalInfo.loggedIn().email()
-                      : Optional.empty();
+                      : applicantPersonalInfo.getType() == ApplicantType.TI_PARTIALLY_CREATED
```

Comment:
That's a good question. I copied this over the logic you had in your working branch, but now I'm realizing maybe there's a reason we don't want emails going to folks that TIs added? I'll check with product on this.

Reply:
Update: We're pretty sure this is a bug. Previously, [emails would be sent](https://github.com/civiform/civiform/blame/bfb86b42f8a945535e76dafea41232f7b5f49ac2/server/app/services/applicant/ApplicantService.java#L499-L511) if there was an email present (which would be true for TI Clients if the TI adds an email). Moreover, when updating the application status for a client added by a TI, their email will show up in the modal, but attempting to send the email results in an error because that case is not accounted for in `updateStatus`.

Reply:
Done.